### PR TITLE
Add FAQ for Element about exploring privacytools.io rooms when it is …

### PR DESCRIPTION
**Pull request by GReagle**
…not your home server

## Description

Resolves: #none 

#### Check List 

- [x] I understand that by not opening an issue about a software/service/similar addition/removal, this pull request will be closed without merging.

- [x] I have read and understand [the contributing guidelines](https://github.com/privacytools/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [x] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: 

* Code repository of the project (if applicable):

### CODE REVIEW

File: pages/services/chat.html [hunk 1]
```diff
@@ -68,6 +68,9 @@ <h4>What happens when I remove a message in Element?</h4>
 <h4>Why do XMPP users keep joining and leaving all the time?</h4>
 <p><a href="https://github.com/matrix-org/matrix-bifrost/issues/63">It's an issue with the XMPP bridge</a>.</p>
 
+<h4>My Matrix home server is not privacytools.io and I want to explore the rooms in the privacytools.io Matrix server.  How do I do that using Element (formerly "Riot")?</h4>
+<p>Click the Explore rooms icon, click the drop down box, click "+ Add a new server ...", type into the box <code>privacytools.io</code> (<strong>not</strong> <code>chat.privacytools.io</chat> which won't work), then click Add.</p>
```
Comment (Mikaela) [inline pages/services/chat.html:72]:
```suggestion
<h4>How do I explore PrivacyTools Matrix room directory from another homeserver using Element?</h4>
<p>Click on the compass icon above the room list, and open the drop down box saying "Matrix-rooms (your homeserver), and from there select the "+ Add a new server ...", and enter <code>privacytools.io</code> (<strong>not</strong> <code>chat.privacytools.io</code> which won't work)into the box and then finally click Add.</p>
```

Comment (GReagle) [reply pages/services/chat.html:72]: Thanks for pointing out that typo or brain-glitch that I made.  Yea it should be /code.

Comment (GReagle) [reply pages/services/chat.html]: I made the change and committed it.  I hope that's the proper procedure.

File: pages/services/chat.html [hunk 2]
```diff
@@ -68,6 +68,9 @@ <h4>What happens when I remove a message in Element?</h4>
 <h4>Why do XMPP users keep joining and leaving all the time?</h4>
 <p><a href="https://github.com/matrix-org/matrix-bifrost/issues/63">It's an issue with the XMPP bridge</a>.</p>
 
+<h4>My Matrix homeserver is not privacytools.io and I want to explore the rooms in the privacytools.io Matrix server.  How do I do that using Element (formerly "Riot")?</h4>
+<p>Click the Explore rooms icon, click the drop down box, click "+ Add a new server ...", type into the box <code>privacytools.io</code> (<strong>not</strong> <code>chat.privacytools.io</code> which won't work), then click Add.</p>
```
Comment (jonaharagon) [inline pages/services/chat.html:72]:
```suggestion
<h4>I'm not using the PrivacyTools homeserver, can I find PrivacyTools rooms in Element?</h4>
<p>Yes! Open the room explorer (compass icon on Element desktop), click the server selection drop down next to the search bar, click "Add a new server", type <code>privacytools.io</code>, and click Add. You should then be able to view and search through rooms that the PrivacyTools server is aware of. Note that not every room hosted on PrivacyTools is affiliated with or moderated by the PrivacyTools team.</p>
```

**Pull request merged**